baseapp/baseapp.go

```diff
@@ -7,11 +7,15 @@ import (
 	"runtime/debug"
 
 	"github.com/golang/protobuf/proto"
+
```

Comment:
Lets just group all nonstandard non-cosmos dependencies together.

Reply:
Okay, cool. 

So, the grouping is:
* go dependencies
* all random dependencies (protobuf, pkg, ...)
* all tendermint dependencies
* all cosmos dependencies

Essentially we are grouping by github organisation.